usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_award.md

```diff
@@ -0,0 +1,180 @@
+## [Spending by Award](#spending-by-award)
+**Route:** `/api/v2/search/spending_by_award/`
+
+**Method:** `POST`
+
+This route takes award filters and fields, and returns the fields of the filtered awards.
+
+### Request
+field: Defines what award variables are returned.
+
+filter: Defines how the awards are filtered.  The filter object is defined here.
```

Comment:
This parameter is actually called `filters` (plural).

Reply:
Indicate that filters are compound (AND operations).

Reply:
Done